src/python/dicomifier/dicom_to_nifti/siemens.py

```diff
@@ -37,10 +37,6 @@ def parse_ascconv(mr_phoenix_protocol):
         MrPhoenixProtocol field stored in the CSA headers.
     """
     
-    ascconv = re.search(
-            b"### ASCCONV BEGIN ###\s*(.*?)\s*### ASCCONV END ###", 
-            mr_phoenix_protocol, flags=re.DOTALL
-        ).group(1)
```

Comment:
This fix looked weird until I noticed there was a bug in the current code 😄 . Please revert this chunk and rebase on 47c3b90.

Reply:
Done. This fix is not needed any more.